source/postprocess/visualization/volume_of_fluid_values.cc

```diff
@@ -0,0 +1,201 @@
+      {
+        prm.enter_subsection("Postprocess");
+        {
+          prm.enter_subsection("Visualization");
+          {
+            prm.enter_subsection("Volume of Fluid");
+            {
+              prm.declare_entry("Include internal reconstruction contour", "false",
+                                Patterns::Bool (),
+                                "Include the internal level set data use to save reconstructed interfaces");
+
+              prm.declare_entry("Include normals", "false",
+                                Patterns::Bool (),
+                                "Include internal normal data in output (DEBUG)");
+            }
+            prm.leave_subsection();
+          }
+          prm.leave_subsection();
+        }
+        prm.leave_subsection();
+      }
+
+
+      template <int dim>
+      void
+      VolumeOfFluidValues<dim>::parse_parameters (ParameterHandler &prm)
+      {
+        prm.enter_subsection("Postprocess");
+        {
+          prm.enter_subsection("Visualization");
+          {
+            prm.enter_subsection("Volume of Fluid");
+            {
+              include_contour = prm.get_bool("Include internal reconstruction contour");
+              include_normal = prm.get_bool("Include normals");
+
+              for (unsigned int f=0; f<this->get_volume_of_fluid_handler().get_n_fields(); ++f)
+                {
+                  std::string field_name = this->get_volume_of_fluid_handler().name_for_field_index(f);
+                  volume_of_fluid_names.push_back("volume_fraction_"+field_name);
+                  interpretations.push_back(DataComponentInterpretation::component_is_scalar);
+
+                  if (include_contour)
+                    {
+                      volume_of_fluid_names.push_back("volume_of_fluid_contour_"+field_name);
+                      interpretations.push_back(DataComponentInterpretation::component_is_scalar);
+                    }
+
+                  if (include_normal)
+                    {
+                      for (unsigned int i=0; i<dim; ++i)
+                        {
+                          volume_of_fluid_names.push_back("volume_of_fluid_interface_normal_"+field_name);
+                          interpretations.push_back(DataComponentInterpretation::component_is_part_of_vector);
+                        }
+                    }
+                }
+            }
+            prm.leave_subsection();
+          }
+          prm.leave_subsection();
+        }
+        prm.leave_subsection();
+      }
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      ASPECT_REGISTER_VISUALIZATION_POSTPROCESSOR(VolumeOfFluidValues,
+                                                  "volume of fluid values",
+                                                  "A visualization output object that outputs the  volume_of_fluid data."
```

Comment:
I think it is not clear what `volume_of_fluid data` means in this context. What about `A visualization output object that outputs the volume fraction and optionally a level set field and the interface normal vectors of volume of fluid fields.`